avm/res/network/private-endpoint/main.bicep

```diff
@@ -174,9 +174,9 @@ output name string = privateEndpoint.name
 output location string = privateEndpoint.location
 
 @description('The group Id for the private endpoint Group.')
-output groupId string = !empty(privateEndpoint.properties.manualPrivateLinkServiceConnections)
-  ? privateEndpoint.properties.manualPrivateLinkServiceConnections[0].properties.groupIds[0]
-  : privateEndpoint.properties.privateLinkServiceConnections[0].properties.groupIds[0]
+output groupId array = !empty(privateEndpoint.properties.manualPrivateLinkServiceConnections)
```

Comment:
I take it it can still only be one - or none? 
If so I'll suggest an alternative output that preserves the string but will await your response :)

Reply:
For instance?

Reply:
Testing your suggestion to remain output as string:

```bicep
@description('The group Id for the private endpoint Group.')
output groupId string = !empty(privateEndpoint.properties.manualPrivateLinkServiceConnections)
  ? privateEndpoint.properties.manualPrivateLinkServiceConnections[0].properties.?groupIds[0] ?? ''
  : privateEndpoint.properties.privateLinkServiceConnections[0].properties.?groupIds[0] ?? ''
  ```

Reply:
which results in 

<img width="1565" alt="image" src="https://github.com/Azure/bicep-registry-modules/assets/19261257/7c96825a-4fee-4a7f-ae5f-968d8b86d1d9">

Reply:
This works: 

```bicep
output groupId string = !empty(privateEndpoint.properties.manualPrivateLinkServiceConnections) && length(privateEndpoint.properties.manualPrivateLinkServiceConnections[0].properties.?groupIds) > 0
  ? privateEndpoint.properties.manualPrivateLinkServiceConnections[0].properties.?groupIds[0] ?? ''
  : !empty(privateEndpoint.properties.privateLinkServiceConnections) && length(privateEndpoint.properties.privateLinkServiceConnections[0].properties.?groupIds) > 0
      ? privateEndpoint.properties.privateLinkServiceConnections[0].properties.?groupIds[0] ?? ''
      : ''
 ```

Reply:
Hey @segraef, maybe using the `first()` function instead would have worked. Anyhow, not need to dwell too much. Thanks for the update, will resolve